lib/mws/fulfillment_outbound_shipment/client.rb

```diff
@@ -13,7 +13,8 @@ module FulfillmentOutboundShipment
     # on when they were fulfilled and by the fulfillment method associated with
     # them.
     class Client < ::Peddler::Client
-      path '/FulfillmentOutboundShipment/2010-10-01'
+      version '2010-10-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.